smart_open/tests/test_s3.py

```diff
@@ -1,21 +1,21 @@
 # -*- coding: utf-8 -*-
-import logging
 import gzip
 import io
+import logging
 import os
-import uuid
 import unittest
+import uuid
+from io import TextIOWrapper
```

Comment:
Do we really need this import? We've already imported io, we can use io.TextIOWrapper.

Reply:
done :)